katalog/cerebro/cerebro.yml

```diff
@@ -26,7 +26,7 @@ spec:
     spec:
       containers:
         - name: cerebro
```

Comment:
Missing liveness/readiness probe, are there any endpoints we can use for probing?